src/sync/impl/sync_file.cpp

```diff
@@ -180,20 +193,49 @@ std::string file_path_by_appending_extension(const std::string& path, const std:
     return buffer;
 }
 
+std::string create_timestamped_template(const std::string& prefix, int wildcard_count)
+{
+    constexpr int WILDCARD_MAX = 20;
+    constexpr int WILDCARD_MIN = 6;
+    // Clamp
+    wildcard_count = (wildcard_count > WILDCARD_MAX ? WILDCARD_MAX : wildcard_count);
+    wildcard_count = (wildcard_count < WILDCARD_MIN ? WILDCARD_MIN : wildcard_count);
```

Comment:
Since we don't have [`std::clamp`](http://en.cppreference.com/w/cpp/algorithm/clamp) yet, I'd probably write this as:
```c++
wildcard_count = std::min(WILDCARD_MAX, std::max(WILDCARD_MIN, wildcard_count));
```